### DISCUSSION TITLE
An astro website to share my replicad models

### DISCUSSION THREAD

**Discussion by sgenoud** [sgenoud/replicad#154] [Show and tell]
I wanted to have a place where I could just throw all the different models that I build - an easy place to link to, just for my models.

https://models.sgenoud.com/

I went a bit too far with the way open graph images are made - but I think it is cool:

<img src="https://models.sgenoud.com/posts/hanger-hook.png" />

I played a little bit with astro and forked a model - I'd be happy to help if someone want to make their own as well!

**Comment by lf94** [comment]
Very cool! I'd love to do the same but use gltf to generate the images. Could maybe even create 'how to print and assemble' books somehow.
